Header header
int32 fromX
int32 toX
int32 fromY
int32 toY
int32 leanTrackingState
Lean lean
bool isRestricted
bool isTracked
uint64 trackingId
int32 clippedEdges
bool engaged
float32 handRightConfidence
int32 handRightState
float32 handLeftConfidence
int32 handLeftState
Appearance appearance
Activities activities
Expressions expressions
JointOrientationAndType[] jointOrientations
JointPositionAndState[] jointPositions
